kserve/runtimes/24.05-llama3-70b-instruct.yaml

```diff
@@ -11,7 +11,7 @@ spec:
   containers:
   - env:
     - name: NIM_CACHE_PATH
-      value: /mnt/models/cache
+      value: /opt/nim/.cache
```

Comment:
The nvidia-nim-pvc will be mounted to the /mnt/models directory. 
If NIM_CACHE_PATH=/opt/nim/.cache is set here, the nvidia-nim-pvc will not work？

Reply:
@Syulin7 `/opt/nim/.cache` is the right one for NIM_CACHE_PATH 
I saw you opened an issue about `/mnt/model` 
NIM with Kserve need storageClass or hostPath that has `ReadWriteMany`

Reply:
Why is /opt/nim/.cache correct? This value was working fine at /opt/nim/cache and I think that is more correct in the KServe world as this is where KServe will dynamically mount the volumes into. Unless we want to bypass KServes method for PVC mounting (which I assume we don't pending resolution of a few bugs) I think this should remain as-is.

Reply:
@supertetelman it was failed to run when we tried to deploy and as per the docs it should change to `/opt/nim/.cache`
 https://docs.nvidia.com/nim/large-language-models/latest/getting-started.html#run-inference

Reply:
It likely failed to run due to https://github.com/kserve/kserve/issues/3687, setting this to `/tmp` works fine for testing and setting it to anything but `/mnt/models` will invalidate the use of a cache/shared volume as this is the only location KServe mounts volumes with the default behaviors.